docs/001_develop/03_client-capabilities/022_framework-integration/index.mdx

```diff
@@ -11,7 +11,106 @@ tags:
 sidebar_position: 22
 ---
 
-Coming soon...
+# Framework integration
+
+Foundation UI is web components based library. This means it can work alongside front end Frameworks such as React and Angular.
+
+This section of our documentation will guide you in how to get Foundation UI working with your existing or preferred stack.
+
+As Foundation UI is based on web components, the components need to be registered in the dom and can then be interacted by setting attributes, properties and listening to events in the same way as any HTML element.
+
+We recommend using our Genesis Create tool to create an Angular or React starter web application to see how the framework is integrated. If you have an existing application you can use the generated create application as a guide to integrating Foundation UI.
+
+## Registering components
+
+As Foundation UI is based on web components the first step is registering the components in the dom. To do this you to import the required modules and components and register the design system (which applies the styles).
```

Comment:
Bit nitpicky but this is the third time in just a few sentences that you say pretty much the same thing, feels a little odd to read. ALso "DOM"

Reply:
Fair point, will try and think of something here.